packages/imperative/src/config/src/Config.ts

```diff
@@ -179,7 +179,8 @@ export class Config {
                 exists: false,
                 properties: Config.empty(),
                 global: layer === Layers.GlobalUser || layer === Layers.GlobalConfig,
-                user: layer === Layers.ProjectUser || layer === Layers.GlobalUser
+                user: layer === Layers.ProjectUser || layer === Layers.GlobalUser,
+                ignoreErrors: process.argv.includes("--help") || process.argv.includes("--version")
```

Comment:
Should we also handle the case of the `zowe` command being run without any arguments since that prints help?

Reply:
If the plan is to only support the `zowe` command being executed by itself, then I think adding an OR-condition might work.
```
  ||  process.argv[process.argv.length-1] === require.resolve('@zowe/cli')
```
  
---

If we want to handle eevry other command (like `zowe config` or `zowe jobs sumbmit`...
Wouldn't we have to check for whether there is no handler for them?

Reply:
i added `||  process.argv[process.argv.length-1] === require.resolve('@zowe/cli')` on ln 184, lmk if I should do something else @zFernand0 @t1m0thyj